### PR TITLE
issue Fixed #7168

### LINKED ISSUE #7168: Searching in admin panel, works only for default language
### Terms

- [x] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

1.4.3

### Issue Description

Searching and also filtering by products and categories in language different than default locale set in .env file always results in redirection to the default language. 

### Preconditions

It's fresh clone from master branch.

### Steps to reproduce

1. Go to Settings --> Channels
2. In Currencies and locales section choose two locales(f.e English and French), save changes
3. Go to Catalog --> Products
4. Create new product in default channel and english locale
5. Edit created product, set locale to French
6. In product list set locale to French
7. Type french name in searching input -> search

### Expected Result

The product list shows products with a French name

### Actual Result

Product list shows products with locales based on default language set

### Screenshots

image
image
image
image

### Additional context

In summary, searching and filtering is only possible in the default language

### PR DESCRIPTION
**Pull request by vijay-webkul**
## Issue Reference
#7168 FIXED ISSUE

### CODE REVIEW

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue [hunk 1]
```diff
@@ -149,7 +150,11 @@ export default {
                 newParams = `${newParams}&${this.filters[i].column}${condition}=${this.filters[i].val}`;
             }
 
-            this.url = `${this.src}?v=1${newParams}`;
+            this.$nextTick(()=>{
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue:153]: Add space after =>

**Review by Sanjaybhattwebkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue (hunk 1 above)
Comment (vijay-webkul) [reply packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue]: ok, I have done.

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
**Not working.** Please **compile** the **vue file** before pushing the code. run the given **command** 

_**npm run i
npm run prod**_

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
**Not working**, please find the below screenshots.

screenshot_1675690434762



screenshot_1675690468439

**Comment by Sanjaybhattwebkul** [comment]
Closing this pr. new pr is created for this issue. https://github.com/bagisto/bagisto/pull/7269

**Pull request closed without merging**